lookout/style/format/analyzer.py

```diff
@@ -503,17 +512,17 @@ def _group_line_nodes(
         :param rule_winners: List of rule winners.
         :return: 1-based line number and sublists of corresponding items from all input sequences.
         """
-        line_no = None
-        line_items = None
+        line_no = -1
+        line_items = []
         result = []
         for (yi, y_predi, vnode_y, winner) in zip(y, y_pred, vnodes_y, rule_winners):
             if winner < 0:
                 continue
-            if vnode_y.start.line != line_no:
+            if vnode_y.end.line != line_no:
```

Comment:
We should include newlines to the next line, not to the previous one. Because if you change ` ⏎␣⁺␣⁺` to ` ⏎` it affects the next line.